quickstart/aspnet-core/infra/core/ai/cognitiveservices.bicep

```diff
@@ -0,0 +1,42 @@
+metadata description = 'Creates an Azure Cognitive Services instance.'
```

Comment:
This PR is adding a huge amount of bicep. Who's going to maintain it? Is this the minimum amount needed to demonstrate Redis concepts?